### PR TITLE
docker: add custom bitcoin conf

### PR DESCRIPTION
**Pull request by PurpleTimez**
This is not tested on infra yet, though that way one can have more data collection after channel jamming tries.

### CODE REVIEW

File: setup/Dockerfile [hunk 1]
```diff
@@ -15,8 +15,13 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 
 # Create bitcoin conf so that bitcoin-cli just works.
-RUN mkdir -p ~/.bitcoin && \
-    echo "rpcconnect=bitcoind.warnet-armada" >> ~/.bitcoin/bitcoin.conf && \
+RUN mkdir -p ~/.bitcoin
+
+# Copy in bitcoin conf that is a placeholder to custom bitcoin conf
+COPY bitcoin.conf ~/.bitcoin/conf
```
Comment (pinheadmz) [inline setup/Dockerfile:21]:
```suggestion
COPY bitcoin.conf ~/.bitcoin/bitcoin.conf
```

**Review by pinheadmz — COMMENTED** [review]
I think you are missing a template bitcoin.conf file so docker has something to copy in. If you do that you mine as well remove all the `echo` statements too right>

Also, how does this help collect data?

**Comment by carlaKC** [comment]
Closing PR - value add is unclear.

**Comment by PurpleTimez** [comment]
> value add is unclear.

@carlaKC 

actually this is also let to you tweak all bitcoind mempool settings (e.g `maxmempool`) to inflate mempool feerate groups at the attacker advantage and e.g fulfill HTLCs buckets faster (see eclair #2330 as mitigation example)

do you feel cross-layer tricks to boost channel jamming attacks efficiency are too much out of the box for now ?

**Comment by carlaKC** [comment]
> actually this is also let to you tweak all bitcoind mempool settings 

This PR only impacts the _attacking_ node's `bitcoin.conf`, change would need to be applied to warnet graph file to update the network of bitcoin nodes.

**Comment by PurpleTimez** [comment]
> This PR only impacts the attacking node's bitcoin.conf, change would need to be applied to warnet graph file to update the network of bitcoin nodes.

Ok boss.

**Pull request closed without merging**